### DISCUSSION TITLE
URL with or without ending slash

### DISCUSSION THREAD

**Discussion by toto975** [ether/etherpad-lite#5673] [Q&A]
I've installed etherpad under Windows IIS.

https://my.domain/tableau-blanc/ (with ending slash) is OK, all js, css and images are downloaded HTTP status 200

https://my.domain/tableau-blanc (without the ending slash) is KO, all js, css and images aren't downloaded HTTP status 404
The link to the resource is https://my.domain/static/js/vendors/jquery.js
It must be https://my.domain/tableau-blanc/static/js/vendors/jquery.js

How to display all without the ending slash ?

**Comment by SamTV12345** [comment]
If the first solution works. Why do you want to choose the other one. You need to have a slash somewhere for the filepath. So my guess is that if there is no slash it just discards the directory. Can you setup IIS to redirect from  to / ?
